src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -26,27 +26,40 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
   setPendingForm
 }) => {
   const [ tabIndex, setTabIndex ] = React.useState(0);
+  const [ disabledValue, setdisabledValue ] = React.useState<boolean>(false);
+  const [ requiredConditionField, setRequiredConditionField ] = React.useState<string>("");
+  const [ requiredConditionFieldBoolean, setRequiredConditionFieldBoolean ] = React.useState<string>("");
```

Comment:
Constant name refers to boolean but type is string, that is bit confusing, rename this.

Reply:
Fill be fixed in visibility PR